docs/get-invited-and-onboard-to-techpass.md

```diff
@@ -13,22 +13,20 @@ Users who need a TechPass account but do not have a WOG account or a non-SE GSIB
 
 You need the following to get invited to TechPass and complete the onboarding:
 
-- Your organisational email address provided by your organisation.
-- A standard mailbox for your organisational email address. TechPass does not support email accounts which do not have an inbox, such as LiteMail accounts. If you use such an email account, upgrade it to a standard mailbox before requesting for TechPass.
-- Before you onboard, ensure you have received the TechPass onboarding email and is still valid.
+- Organisational email address with a standard mailbox (not LiteMail).
+- Valid TechPass onboarding email.
 
 ?> TechPass onboarding email is valid for 30 days. If you do not onboard to TechPass within this 30 days, we will terminate your TechPass account, and you need to sign up again.
 
 
-> **Tip**: Click the triangle to view the instructions to complete each step.
 
 ## Step 1. Request for TechPass account
 
-<details><summary style="font-size:20px;font-weight:bold">Get invited to TechPass account and request for SEED (optional)</summary>
+<details data-is-open="true" data-is-size="medium"> <summary>Get invited to TechPass account and request for SEED (optional)</summary>
 
 1. Contact your project manager or the reporting officer to request for TechPass account and SEED provisioning(optional).
 
-!> You need SEED provisioning to access SGTS and GCC 2.0 resources via an Internet Device.
+!> To access service such as SGTS and GCC 2.0 resources through an Internet Device, you need to onboard that device to SEED.
```

Comment:
_services_ not service. 
_that_ sounds incorrect. the?

Reply:
good catch, thanks!